Makefile

```diff
@@ -331,9 +331,6 @@ docker_lite_all: docker_lite $(DOCKER_LITE_TARGETS)
 docker_lite:
 	${call build_docker_image,docker/lite/Dockerfile,vitess/lite}
 
-docker_local:
-	${call build_docker_image,docker/local/Dockerfile,vitess/local}
-
 docker_run_local:
 	./docker/local/run.sh
```

Comment:
Let's also remove this please, we do not need it anymore.

Reply:
this = `docker_run_local`

Reply:
removed this too!